pkgs/servers/gerbera/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv, fetchgit
+, cmake, pkg-config
+# required
+, libupnp, libuuid, pugixml, libiconv, sqlite, zlib, spdlog, fmt
+, pkgs
+# options
+, enableDuktape ? true
+, enableCurl ? true
+, enableTaglib ? true
+, enableLibmagic ? true
+, enableLibmatroska ? true
+, enableAvcodec ? false
+, enableLibexif ? true
+, enableExiv2 ? false
+, enableFFmpegThumbnailer ? false
+, enableInotifyTools ? true
+}:
+
+with stdenv.lib;
+let
+  optionOnOff = option: if option then "on" else "off";
+in stdenv.mkDerivation rec {
+  pname = "gerbera";
+  version = "1.5.0";
+
+  src = fetchgit {
+    url = meta.repositories.git;
+    rev = "0c486fc53b097227db2443c738647690635bdf99";
+    sha256 = "sha256:19370aagyqyik5dx3ymr0l4sqgwsa5bv8n25lcdcdg1vm2avq0y1";
+  };
+
+  cmakeFlags = [
+    "-DWITH_JS=${optionOnOff enableDuktape}"
+    "-DWITH_CURL=${optionOnOff enableCurl}"
+    "-DWITH_TAGLIB=${optionOnOff enableTaglib}"
+    "-DWITH_MAGIC=${optionOnOff enableLibmagic}"
+    "-DWITH_MATROSKA=${optionOnOff enableLibmatroska}"
+    "-DWITH_AVCODEC=${optionOnOff enableAvcodec}"
+    "-DWITH_EXIF=${optionOnOff enableLibexif}"
+    "-DWITH_EXIV2=${optionOnOff enableExiv2}"
+    "-DWITH_FFMPEGTHUMBNAILER=${optionOnOff enableFFmpegThumbnailer}"
+    "-DWITH_INOTIFY=${optionOnOff enableInotifyTools}"
+    "-DWITH_SYSTEMD=OFF"
```

Comment:
maybe leave a comment why you disabled this.

Reply:
Yes, ok.

Heads up. I recall it's because it did not compile with it.
As we will generate a systemd service ourselves (based on what the existing mediatomb service is doing and what I did on the gerbera service [2], I considered it's fine without it).

Reply:
> Heads up. I recall it's because it did not compile with it.

No, it compiles fine in the end (just checked back). 
It just does not finish as i did not make the necessary adaptation to the derivation for the systemd part.